get_that_data.py

```diff
@@ -1,41 +1,7 @@
 from src.data.data_fetcher import DataFetcher
 
-
-
-fetcher = DataFetcher("2020", "P")
-print(fetcher.api_starting_url_container)
-print(fetcher.gimmie_data(record_limit=5))
-
-
-#### Need to write this into the other file
-
-# Create instance of data_fetcher class class
-# fec = data_fetcher(starting_url, complete_list)
-
-# # Inizialize get_the_data to work
-# work = get_the_data(fec)
-
-# # Starts looping over all transactions
-# work.gimmie_data()
-
-# # Once complete_list has all transaction we create a pandas dataframe
-# df = pd.DataFrame(
-#     complete_list,
-#     columns=[
-#         "contributor_occupation",
-#         "contributor_employer",
-#         "contributor_city",
-#         "contributor_state",
-#         "contributor_zip",
-#         "party",
-#     ],
-# )
-
-# Saves DataFrame as serialized object, will import this to ML program
-# Add path and uncommet to save pickled data
-# df.to_pickle("./pickled_data.pkl")
-
-# Prints DataFrame, be careful, this can get bigggggg
-# print(df)
-
-# Thanks for reading my Ted Talk
\ No newline at end of file
+fetcher = DataFetcher("2020", "S")
+fetcher.api_starting_url_container
```

Comment:
Does this line change mean what I think it means? O_O

https://napsterinblue.github.io/blog/2018/05/01/why-i-switched-to-python-3/

Reply:
You lost me, I have tried for a day now to figure out what you are talking about and I have no clue whatsoever lol.

Reply:
I was referring to removing the `print()` statement

![image](https://user-images.githubusercontent.com/3505365/101675150-72daa780-3a27-11eb-9f32-cb7e08dcf645.png)

thought the reason you had done that was because it errored out in Python 2, but is standard in Python 3. Can't remember if I ever asked which version you were on.

The article was my way of convincing you to cut over to 3, if you were on 2. Generally-speaking, though, I'm really pleased with how I wrote that and think it does a great job selling/explaining one of the bigger selling points of the way Python 3 is designed

Reply:
I kind of thought that might be what you were referring to, and I just got rid of `print()` because it now saves everything to a csv. But I started on 3, never will ever use 2 lol. But yeah it was a good article and I learned more about `print()` than I had known.